server/routes.js

```diff
@@ -197,6 +201,20 @@ export default app => {
    */
   app.post('/helloworks/callback', helloworks.callback);
 
+  // Mount Hyperwatch API and Websocket
+  if (config.hyperwatch && config.hyperwatch.secret && parseToBoolean(config.hyperwatch.enabled) === true) {
+    // We need to setup express-ws here to make Hyperwatch's websocket works
+    expressWs(app);
+    const hyperwatchPath = '/_hyperwatch';
+    const hyperwatchBasicAuth = expressBasicAuth({
+      users: { [config.hyperwatch.username]: config.hyperwatch.secret },
+      challenge: true,
+      realm: config.hyperwatch.realm,
```

Comment:
`config.hyperwatch.realm` seems to be missing in the config

Reply:
Added as `null` to the default config.